docs/SPIRVRepresentationInLLVM.rst

```diff
@@ -74,6 +75,24 @@ mangled as __spirv_{TypeName}, where {TypeName} is the name of the SPIR-V
 type with "OpType" removed, e.g., OpTypeEvent is mapped to spirv.Event and
 mangled as __spirv_Event.
 
+Address spaces
+--------------
+
+The following
+`SPIR-V storage classes <https://www.khronos.org/registry/spir-v/specs/unified1/SPIRV.html#Storage_Class>`_
+are naturally represented as LLVM IR address spaces with the following mapping:
+
+SPIR-V storage class    LLVM IR address space
+Function                No address space or addrspace(0)
```

Comment:
This doesn't render as a table currently.

It's also nicer to render the keywords as inline literals using ` `` `

Reply:
Which words are keywords? `addrspace(x)`? Or SPIR-V storage classes as well (i.e. `Function`)?

Reply:
I'd suggest both ``addrspace(x)`` and SPIR-V storage classes.

Reply:
Done.